### PR TITLE
Feature/i57/create basic routing

### PR DESCRIPTION
**Pull request by Rustyiller**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,5 +12,15 @@
...
+  },
+  "devDependencies": {
+    "cross-env": "^5.2.0",
+    "eslint": "^5.5.0",
+    "eslint-config-prettier": "^3.0.1",
+    "eslint-plugin-prettier": "^2.6.2",
+    "jest": "^24.5.0",
+    "prettier": "^1.14.2",
+    "sinon": "^6.3.3",
+    "sqlite3": "^4.0.9"
```
Comment (kulaszewicz) [inline package.json]: sqlite and sinon are not needed

File: app.js [hunk 2]
```diff
@@ -21,21 +22,22 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', indexRouter);
 app.use('/users', usersRouter);
```
Comment (kulaszewicz) [inline app.js]: all user related stuff need to be deleted

**Review by kulaszewicz — CHANGES_REQUESTED** [review]
Added some comments about what to change

**Review by kulaszewicz — APPROVED** [review]
Looks good

**Review by sj-lt — APPROVED** [review]

**Pull request merged**